tests/python/test_python_frontend.py

```diff
@@ -701,7 +701,7 @@ def fusion_func(fd: FusionDefinition):
             fd.add_output(t1, alias_input=t0)
             fd.add_output(t2)
 
-        out_tensors, _ = self.exec_nvfuser(fusion_func, in_tensors)
+        out_tensors, _ = self.exec_nvfuser(fusion_func, in_tensors, is_clonable=True)
```

Comment:
Should we just make `is_clonable=True` the default and mark the ones that currently are not clonable?

Reply:
With this PR, there are roughly 78 failing and 35 passing. Most of the 35 passing tests overlap with the binary opinfo tests. I need a test specific for alias testing.